### ISSUE TITLE
Ctrl+arrow navigates results rows even when editing text

### ISSUE THREAD

**Issue by BobKnip** [dbeaver/dbeaver#2715]
In DBeaver 4.3.1, Ctrl+Left and Ctrl+Right always navigate to Previous Row / Next Row in the Results viewer, even when actively editing text. 
- The same faulty behavior occurs while editing Results filter. 
- The same faulty behavior occurs for Ctrl+Shift+arrow: First/Last Row instead of Select Previous/Next Word respectively. 
- This still happens after hiding the Results viewer. 
- This happens regardless whether the Results viewer shows the Grid or a single Record. 
- N.B.: When the first/last result is selected or selected, one direction of text editing shortcuts does work: Previous/Next Word respectively. 

The only workaround I can come up with (without changing default keyboard shortcuts) is to execute invalid or commented out SQL, so there are no result rows to navigate between. Then navigating text with Ctrl+Left = Previous Word / Ctrl+Right = Next Word does work.

**Comment by serge-rider** [comment]
I can't reproduce that.
What is your OS?

Also you could try to reset workspace (https://github.com/dbeaver/dbeaver/wiki/Reset-workspace).  Please report about results.

**Comment by BobKnip** [comment]
Thank you for the feedback/tip. 
I prefer to leave resetting the workspace as a last resort. I first tried closing and restarting DBeaver; this seems to restore normal functionality. I closed and restarted DBeaver again to make sure it was not a fluke; it still works correctly again. This surprises me a little, as I had certainly already closed and restarted DBeaver once after the last update (today, from 4.3.0 to 4.3.1, both Community Edition 64 bit installer with JRE). 
For completeness sake: my OS is Win10 Pro 64-bit (fully upgraded/updated as of 2018-01-02).
